api/ExamplePayload.json

```diff
@@ -36,14 +36,14 @@
  	"applicantImportCaretakerInformation": true,
  	"navProgress": [100, 100, 100],
  	"applicantLastName": "asdf",
- 	"applicant-email": "inar@smail.com",
+ 	"applicantEmail": "inar@smail.com",
  	"userfullname": "Inar",
```

Comment:
Some of these are not completely camelCase yet ("userfullname", "mandatorycitizenRegistryhidden") but this is just an example payload - and I think taken from an actual implementation. So possibly we prefer to keep it like this?